main/src/cgeo/geocaching/calculator/CaseCheck.java

```diff
@@ -0,0 +1,25 @@
+package cgeo.geocaching.calculator;
+
+/**
+ * Class used for checking that a value is with in a given range.
+ * This is used to check for upper-case an lower-case letters.
+ */
+public final class CaseCheck {
```

Comment:
Could org.apache.commons.lang3.CharUtils be used instead?

If not: can this be moved as method to an existing Utils class? Maybe TextUtils?

Reply:
I just moved the class out of the CalculatorDialog as whole.

But I will have a look into the package if it fits the needs

Reply:
Moved it as method to TextUtils